src/nvim/os/unix_defs.h

```diff
@@ -2,7 +2,7 @@
 #define NVIM_OS_UNIX_DEFS_H
 
 // Windows doesn't have unistd.h, so we include it here to avoid numerous
-// instances of `#ifdef HAVE_UNISTD_H'.
+// instances of `#ifdef WIN32'.
 #include <unistd.h>
```

Comment:
Aren't we stepping backwards from the goal of https://github.com/neovim/neovim/issues/549, which is to "flatten" the includes instead of centralizing them in a mega-include? 

For #549 we may need to use HAVE_UNISTD_H and explicitly include `<unistd.h>`. We started out trying to pull apart `globals.h` and `vim.h`, but `os_defs.h` is now turning into something similar.

If I'm mistaken please let me know.